test/cmocka/src/math/arithmetic/exponential.c

```diff
@@ -81,10 +92,107 @@ static void test_math_arithmetic_exponential_fixed(void **state)
 	}
 }
 
+static void gen_exp_testvector_linspace_q27(double a, double b, int step_count,
+					    int point, int32_t *value)
+{
+	double fstep = (b - a) / (step_count - 1);
+	double fvalue = a + fstep * point;
+
+	*value = (int32_t)round(fvalue * 134217728.0);
```

Comment:
out of curiosity - that number has a meaning, right? I might be the only one who doesn't recognise it (ok, I'm sure it isn't Pi and it isn't `e`, I'm certain enough about that :-) ) but maybe a comment would be helpful